src/app/utils/BountyUtils.ts

```diff
@@ -80,8 +80,8 @@ const BountyUtils = {
         }
     },
 
-    validateEvergreen(evergreen: boolean, claimLimit: number, gateOrAssign: boolean) {
-        if (evergreen && gateOrAssign) {
+    validateEvergreen(evergreen: boolean, claimLimit: number, gate: boolean) {
+        if (evergreen && gate) {
```

Comment:
I think you have this backwards. You want to allow gates with evergreens, right? Need to fix the validation message also